### ISSUE TITLE
Shadows in rear cockpit

### ISSUE THREAD

**Issue by viktorradnai** [viktorradnai/flightgear-ask21#87]
Looks like the shadows aren't applied to the rear cockpit interior -- or perhaps there is a shadow that should not be there. I've noticed a shadow appearing in the footwell that did not continue into the instructor's cockpit area.

![fgfs-screen-263](https://cloud.githubusercontent.com/assets/4738178/23866350/776d66c2-0810-11e7-8f01-cc93bbb7b44e.png)
![fgfs-screen-264](https://cloud.githubusercontent.com/assets/4738178/23866349/776d0790-0810-11e7-9aa6-d0d15628da54.png)

**Comment by ghost** [comment]
-wrong cubemap is fixed but we still need a possibility for darker shading in the footwell

**Comment by ghost** [comment]
I think we might want to use some irradiance map here, so we can have darker shading in the footwell, but I haven't yet figured out how to do so @gilbertohasnofb do you know that by chance?

**Comment by gilbertohasnofb** [comment]
@D-ECHO I do not know how the irradiance map works but perhaps @wlbragg could point you in the right direction.

**Comment by wlbragg** [comment]
>I think we might want to use some irradiance map here, so we can have darker shading in the footwell,

I'm not sure that is the issue here. You should get a darker footwell simply by using the default model interior.

**Comment by wlbragg** [comment]
After looking at the models, is it because the fuselage model has nothing to indicate that the footwell is in shadow. Does it not recognize the separate interior.ac as casting shadow?

**Comment by wlbragg** [comment]
When making the shadow map you would want the entire model assembled into one piece for one.
As far as the two separate models ask21.ac and interior.ac recognizing the other has influence, I will have to download the repo and test it for myself to wrap my head around it.

**Comment by ghost** [comment]
Thanks @wlbragg for looking at this, but I have to admit I don't really understand what you're writing. With "shadow map" do you mean the black cubemap passed to the interior-model.eff (<texture n="4">) or the AO shadows baked onto the texture?

**Comment by wlbragg** [comment]
>the black cubemap passed to the interior-model.eff 

Yes that is the one, but that still wont get you darker recessed areas. 

I just installed this and it looks reasonable  to me. I didn't notice any misplaced shadows. I think it is fairly light down in the recesses because of the open top.
